src/kagglehub/config.py

```diff
@@ -28,6 +34,7 @@
     "error": logging.ERROR,
     "critical": logging.CRITICAL,
 }
+TRUTHY_VALUES = ["true", "1", "t"]
```

Comment:
Is `t` sometimes used?

Reply:
I have seen this for a few env vars in the past.